modules/services/avizo.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkEnableOption ''
+      avizo, a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway
+    '';
+    config = mkOption {
+      type = (pkgs.formats.ini { }).type;
+      default = { };
+      example = {
+        default = {
+          time = 1.0;
+          y-offset = 0.5;
+          fade-in = 0.1;
+          fade-out = 0.2;
+          padding = 10;
+        };
+      };
+      description =
+        "Attrs that will be converted to .ini avizo configuration file.";
+    };
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.avizo;
+      defaultText = literalExpression "pkgs.avizo";
+      example = literalExpression ''
+        pkgs.avizo.overrideAttrs (final: prev: {
```

Comment:
Are you sure this is right? Typically the `overrideAttrs` argument should be a function that just takes a single argument. Something like `overrideAttrs (oldAttrs: { …})`.

Reply:
Yes, according to dots: https://nixos.org/manual/nixpkgs/stable/#sec-pkg-overrideAttrs it also works in my personal setup